sdks/aperture-js/README.md

```diff
@@ -14,9 +14,9 @@
 
 # Aperture JavaScript SDK
 
-`aperture-js` is an SDK to interact with [Aperture](https://docs.fluxninja.com)
-Agent. It allows flow control functionality on fine-grained features inside
-service code.
+The `aperture-js` SDK provides an easy to way to integrate your js applications
```

Comment:
There is a typo here `an easy to way to` -> `an easy way to`

Reply:
Fixed in next commit.